Sources/SwiftServerHttp/HTTPRequest.swift

```diff
@@ -73,4 +73,82 @@ public enum HTTPMethod: String {
     case MKCALENDAR
     case LINK
     case UNLINK
+  
+    public init(rawValue: String) {
+        switch rawValue {
+            case "DELETE":      self = .DELETE
+            case "GET":         self = .GET
+            case "HEAD":        self = .HEAD
+            case "POST":        self = .POST
+            case "PUT":         self = .PUT
+            case "CONNECT":     self = .CONNECT
+            case "OPTIONS":     self = .OPTIONS
+            case "TRACE":       self = .TRACE
+            case "COPY":        self = .COPY
+            case "LOCK":        self = .LOCK
+            case "MKCOL":       self = .MKCOL
+            case "MOVE":        self = .MOVE
+            case "PROPFIND":    self = .PROPFIND
+            case "PROPPATCH":   self = .PROPPATCH
+            case "SEARCH":      self = .SEARCH
+            case "UNLOCK":      self = .UNLOCK
+            case "BIND":        self = .BIND
+            case "REBIND":      self = .REBIND
+            case "UNBIND":      self = .UNBIND
+            case "ACL":         self = .ACL
+            case "REPORT":      self = .REPORT
+            case "MKACTIVITY":  self = .MKACTIVITY
+            case "CHECKOUT":    self = .CHECKOUT
+            case "MERGE":       self = .MERGE
+            case "MSEARCH":     self = .MSEARCH
+            case "NOTIFY":      self = .NOTIFY
+            case "SUBSCRIBE":   self = .SUBSCRIBE
+            case "UNSUBSCRIBE": self = .UNSUBSCRIBE
+            case "PATCH":       self = .PATCH
+            case "PURGE":       self = .PURGE
+            case "MKCALENDAR":  self = .MKCALENDAR
+            case "LINK":        self = .LINK
+            case "UNLINK":      self = .UNLINK
+            default:            self = .other(rawValue)
+        }
+    }
```

Comment:
@helje5 I'm thinking since HTTPMethod are pretty well-defined, maybe we can use StaticString instead of String here??